cmplibrary/src/main/java/com/sourcepoint/cmplibrary/creation/SpConfigDataBuilder.kt

```diff
@@ -23,6 +24,7 @@ class SpConfigDataBuilder {
     var campaignsEnv: CampaignsEnv = CampaignsEnv.PUBLIC
     var messageTimeout: Long = 5000
     var logger: Logger? = null
+    var gppConfig: GppConfig? = null
 
     operator fun CampaignType.unaryPlus() {
```

Comment:
I actually see no advantage in overloading `unaryPlus` in this case. I think the idea was to make it almost intuitive in terms of readability, but it made it less readable from my point of view.

Here is also the response from the ChatGPT:

> The advantage of overloading the unaryPlus function is that it provides you with the ability to define a custom action that should be performed when the + unary operator is used with an instance of your class. This can be particularly useful in scenarios where you want to create more expressive and intuitive APIs for your classes.

And again, from my point of view, it actually doesn't make it _"more expressive and intuitive"_.

But since it works and it is the part of our public API, my proposition is to leave it **as is** for now.